infra/src/test/kotlin/fi/fta/geoviite/infra/ui/testgroup2/LinkingTestUI.kt

```diff
@@ -131,13 +131,10 @@ class LinkingTestUI @Autowired constructor(
         val geometryTrackEndPoint = geometryAlignment.elements.last().end
 
         val (locationTrackStartPoint, locationTrackEndPoint) = alignmentDao.fetch(
-            locationTrackDao.fetch(
-                originalLocationTrack
-            ).alignmentVersion!!
-        ).let { alignment ->
-            alignment.start!! to alignment.end!!
-        }
+            locationTrackDao.fetch(originalLocationTrack).alignmentVersion!!
+        ).let { alignment -> alignment.start!! to alignment.end!! }
 
+        E2ETrackLayoutPage.finishLoading()
```

Comment:
Mmh. Mietin että sen sijasta että lisäilisi näitä eksplisiittisesti testeihin, niin pitäisikö tämä vaan sisällyttää suoraan kaikkiin karttakliksuttelufunktioihin? Tai siis ainakin omasta mielestäni ne voisivat hyvin toimia siten, että jos kartta ei ole valmis, niin odotetaan ja vasta sen jälkeen koitetaan tehdä mikä operaatio ikinä haluttiinkaan tehdä

Reply:
Joo ihan hyvä idea. Siirrän sinne.